.run/S3IntegrationApplication.run.xml

```diff
@@ -0,0 +1,20 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="S3IntegrationApplication" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
+    <option name="ACTIVE_PROFILES" value="local" />
+    <module name="refarch-s3-integration-rest-service (1)" />
```

Comment:
Is this really correct? The `(1)` at the end looks kind of strange to me.

Reply:
Done